draft-dekater-scion-dataplane.md

```diff
@@ -1342,20 +1355,101 @@ This section describes the steps that a SCION egress border router must perform
 
 # Security Considerations
 
-TODO security considerations.
+This section describes the possible security risks and attacks that SCION's data plane may be prone to, and how these attacks may be mitigated. It first discusses security risks that pertain to path authorization, followed by a section on all other data-plane security risks.
+
+**Note:** This section only discusses SCION data plane- and forwarding-specific security considerations. For security considerations related to the SCION control-plane PKI, see {{I-D.scion-cppki}}. {{I-D.scion-cp}} includes security considerations that concern the SCION control plane and routing.
+
+
+## Security Risks in Regard to Path Authorization
+
+A central property of the SCION path-aware data plane is path authorization. Path authorization guarantees that data packets always traverse the network along paths segments authorized in the control plane by all on-path ASes. This section discusses how an adversary may attempt to violate the path-authorization property, as well as SCION's answers to these attacks. The following attacks are imaginable:
+
+- Crafting unauthorized hop fields (see [](#crafting)),
+- modifying the validity period of a path segment (see [](#mod-validity)), or
+- path splicing, that is, constructing an unauthorized segment by combining hop fields of different valid segments (see [](#path-splicing)).
+
+
+### Crafting Unauthorized Hop Fields {#crafting}
+
+Hop fields are protected with MACs. To be able to craft and misuse unauthorized hop fields, an adversary must therefore either determine the MAC's key or try to break the MAC directly.
```

Comment:
Is 'crack' a better work than 'break' here (and further down the text)?

Reply:
I'd say "crack the underlying MAC algorithm"